ACCWebAPI/Controllers/IncomingInvoiceController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using ACC.ViewModel.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using ACCDataModel.Kostenrechnung;
+
+namespace ACCWebAPI.Controllers
+{
+    [Authorize]
+    [ApiController]
+    [Route("incoming_invoices")]
+    public class IncomingInvoiceController : ControllerBase
+    {
+        private readonly ILogger<IncomingInvoiceController> _logger;
+        private readonly IACCService _accService;
+
+        public IncomingInvoiceController(ILogger<IncomingInvoiceController> logger, IACCService accService)
+        {
+            _logger = logger;
+            _accService = accService;
+        }
+
+        [HttpGet("")]
+        public async Task<IActionResult> Index()
+        {
+            try
+            {
+                IEnumerable<Eingangsrechnung> result = await _accService.GetEingangsrechnungenAsync();
+
+                return Ok(result);
+
+            }
+            catch (Exception ex)
+            {
+                return StatusCode(500, ex.Message);
+            }
+        }
+
+        [HttpPost("")]
+        public async Task<IActionResult> Store(Eingangsrechnung eingangsrechnung)
```

Comment:
Lets also rename this method to meet the .Net naming guidelines: PostInvoiceToDB()